d2bs/kolbot/libs/modules/AutoEquip.js

```diff
@@ -363,7 +102,7 @@
 			if (item.identified) {
 
 				// no point in wanting items we cant equip
-				if (item.getStat(sdk.stats.Levelreq) > me.getStat(sdk.stats.Level) || item.dexreq > me.getStat(sdk.stats.Dexterity) || item.strreq > me.getStat(sdk.stats.Strength)) {
+				if (item.getStat(sdk.stats.Levelreq) > this.reference.getStat(sdk.stats.Level) || item.dexreq > this.reference.getStat(sdk.stats.Dexterity) || item.strreq > this.reference.getStat(sdk.stats.Strength)) {
```

Comment:
Here we need to add extra check if item what we want replace doesn't give us bonus str/dex so we end up beign unable to equip new item.

When extra bonus stats from items are super handy i suggest to go with raw stats.